ERROR    root:QAAppAllure.py:165 ❌ Failed to accept cookies.
ERROR    root:QAAppAllure.py:165 ❌ Failed to accept cookies.
ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiNzQzMTcyNDY2LjE3NDcxMzE5OTAifSwiYW5vbklkIjoiMTI5MzI2YzNiNjMyZDYyNSJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL21vZGVscy9zdXYvZXFlL292ZXJ2aWV3Lmh0bWw%2FaW50ZXJuYWxfdGVzdD10cnVlIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2NCIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjEzMzUxMDA1Nzk2ODczNTk0IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=136.0.7103.59)
Stacktrace:
#0 0x560dfc81d7fa <unknown>
#1 0x560dfc2c1e90 <unknown>
#2 0x560dfc2a9156 <unknown>
#3 0x560dfc2a7a6a <unknown>
#4 0x560dfc2a81cf <unknown>
#5 0x560dfc2a8124 <unknown>
#6 0x560dfc2c611f <unknown>
#7 0x560dfc36b3ca <unknown>
#8 0x560dfc339242 <unknown>
#9 0x560dfc35f135 <unknown>
#10 0x560dfc339013 <unknown>
#11 0x560dfc305b3b <unknown>
#12 0x560dfc3067a1 <unknown>
#13 0x560dfc7e2b9b <unknown>
#14 0x560dfc7e6a8a <unknown>
#15 0x560dfc7ca912 <unknown>
#16 0x560dfc7e7604 <unknown>
#17 0x560dfc7af74f <unknown>
#18 0x560dfc80b678 <unknown>
#19 0x560dfc80b856 <unknown>
#20 0x560dfc81c666 <unknown>
#21 0x7f108b09caa4 <unknown>
#22 0x7f108b129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyOTQ2IiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJpdF9JVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9lcWUvb3ZlcnZpZXcuaHRtbD9pbnRlcm5hbF90ZXN0PXRydWUiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjY0In0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI3NDMxNzI0NjYuMTc0NzEzMTk5MCJ9LCJhbm9uSWQiOiIxMjkzMjZjM2I2MzJkNjI1In0sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6MX0sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnsiUHJvZHVjdCI6eyJfaWQiOiIyOTQ2IiwiZGltZW5zaW9ucyI6eyJCYXVtdXN0ZXJTaG9ydCI6WyIyOTQ2Il19fX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMDgxNjYwMjI0NzIzMTE0NjMifQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=136.0.7103.59)
Stacktrace:
#0 0x560dfc81d7fa <unknown>
#1 0x560dfc2c1e90 <unknown>
#2 0x560dfc2a9156 <unknown>
#3 0x560dfc2a7a6a <unknown>
#4 0x560dfc2a81cf <unknown>
#5 0x560dfc2a8124 <unknown>
#6 0x560dfc2c611f <unknown>
#7 0x560dfc36b3ca <unknown>
#8 0x560dfc339242 <unknown>
#9 0x560dfc35f135 <unknown>
#10 0x560dfc339013 <unknown>
#11 0x560dfc305b3b <unknown>
#12 0x560dfc3067a1 <unknown>
#13 0x560dfc7e2b9b <unknown>
#14 0x560dfc7e6a8a <unknown>
#15 0x560dfc7ca912 <unknown>
#16 0x560dfc7e7604 <unknown>
#17 0x560dfc7af74f <unknown>
#18 0x560dfc80b678 <unknown>
#19 0x560dfc80b856 <unknown>
#20 0x560dfc81c666 <unknown>
#21 0x7f108b09caa4 <unknown>
#22 0x7f108b129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6Ijc0MzE3MjQ2Ni4xNzQ3MTMxOTkwIn0sImFub25JZCI6IjEyOTMyNmMzYjYzMmQ2MjUifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2VxZS9vdmVydmlldy5odG1sP2ludGVybmFsX3Rlc3Q9dHJ1ZSIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjQiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoxfSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI4NDgyMTQ5NzY0NTc5Mjg4IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=136.0.7103.59)
Stacktrace:
#0 0x560dfc81d7fa <unknown>
#1 0x560dfc2c1e90 <unknown>
#2 0x560dfc2a9156 <unknown>
#3 0x560dfc2a7a6a <unknown>
#4 0x560dfc2a81cf <unknown>
#5 0x560dfc2a8124 <unknown>
#6 0x560dfc2c611f <unknown>
#7 0x560dfc36b3ca <unknown>
#8 0x560dfc339242 <unknown>
#9 0x560dfc35f135 <unknown>
#10 0x560dfc339013 <unknown>
#11 0x560dfc305b3b <unknown>
#12 0x560dfc3067a1 <unknown>
#13 0x560dfc7e2b9b <unknown>
#14 0x560dfc7e6a8a <unknown>
#15 0x560dfc7ca912 <unknown>
#16 0x560dfc7e7604 <unknown>
#17 0x560dfc7af74f <unknown>
#18 0x560dfc80b678 <unknown>
#19 0x560dfc80b856 <unknown>
#20 0x560dfc81c666 <unknown>
#21 0x7f108b09caa4 <unknown>
#22 0x7f108b129c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:19 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'